api/src/main/java/org/motechproject/nms/api/web/contract/mobileAcademy/CourseResponse.java

```diff
@@ -0,0 +1,43 @@
+package org.motechproject.nms.api.web.contract.mobileAcademy;
+
+import org.motechproject.nms.api.web.contract.mobileAcademy.course.Chapter;
+
+import java.util.List;
+
+/**
+ * Response object for get course API 2.2.2
+ */
+public class CourseResponse {
+
+    // course name
+    private String name;
+
+    // course version in epoch datetime format
+    private int courseVersion;
```

Comment:
Yes. I was planning on doing the validation in the next checkin where the MA service domain object gets mapped to the MA API response object.